server/channels/app/busy_test.go

```diff
@@ -119,11 +119,14 @@ func compareBusyState(t *testing.T, busy1 *Busy, busy2 *Busy) bool {
 // ClusterMock simulates the busy state of a cluster.
 type ClusterMock struct {
 	Busy *Busy
+	t    *testing.T
```

Comment:
Where does this get set?

Reply:
```go
func TestSendClusterMessage(t *testing.T) {
    busy := &Busy{}
    clusterMock := &ClusterMock{
        Busy: busy,
        t:    t,
    }

    msg := &model.ClusterMessage{
        Data: []byte(`{"some": "data"}`),
    }

    clusterMock.SendClusterMessage(msg)
}
```

May I add something like this?

Reply:
There are test like `TestBusySet` which use a `ClusterMock`. You need to add the `testing.T` there.

https://github.com/mattermost/mattermost/blob/63c97f5a6d8f55eb7583ad975ce12a20d6478273/server/channels/app/busy_test.go#L18-L19